ColorPatterns/Scenes/Game/GameViewModel.swift

```diff
@@ -54,7 +55,7 @@ private extension GameViewModel {
         }
     }
 
-    //TODO: Remove prints when not needed
+    // TODO: Remove prints when not needed
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/c07cd9a3b7a63e61e86332aeb4fa4cabf8b81c17/ColorPatterns/Scenes/Game/GameViewModel.swift#L58"></span>TODOs should be resolved (Remove prints when not needed).<br />
<code>todo</code> <code>GameViewModel.swift:58</code></td>
    </tr>
  </tbody>
</table>